compose-builder/Makefile

```diff
@@ -36,7 +36,7 @@ define OPTIONS
  - zmq-bus mqtt-bus mqtt-broker -
  - taf-secty taf-no-secty taf-perf taf-perf-no-secty -
  - ds-bacnet ds-camera ds-grove ds-modbus ds-mqtt ds-rest ds-snmp ds-virtual ds-llrp ds-coap ds-gpio -
- - asc-http asc-mqtt asc-sample as-llrp -
+ - asc-http asc-mqtt asc-sample as-llrp asc-external-mqtt-trigger -
```

Comment:
Please update the README for this new option.

Reply:
Changed the option name from asc-external-mqtt-trigger to asc-ex-mqtt and updated the README file.

Reply:
Nice. I was going to suggest to shorten the name. :-)